src/python/pants/goal/run_tracker.py

```diff
@@ -387,15 +390,51 @@ def store_stats(self):
     if target_data:
       run_information['target_data'] = ast.literal_eval(target_data)
 
+    def get_option(scope_or_scope_with_flag, flag=None):
```

Comment:
Better to use the word 'option' here, not 'flag'. The value of the option may not come from a flag. In fact it usually won't.

Reply:
Done